packages/pn-pa-webapp/src/api/external-registries/External-registries.api.ts

```diff
@@ -19,4 +22,33 @@ export const ExternalRegistriesAPI = {
     apiClient
       .get<Array<Party>>(GET_PARTY_FOR_ORGANIZATION(organizationId))
       .then((response: AxiosResponse<Array<Party>>) => response.data[0]),
+
+
+  getInstitutions: (): Promise<Array<PartyEntity>> =>
+    apiClient
+      .get<Array<Institution>>(GET_INSTITUTIONS())
+      .then((response: AxiosResponse<Array<Institution>>) => response.data.map((institution) => (
+        {
+          id: institution.id,
+          name: institution.description,
+          productRole: institution.userProductRoles[0],
+          logoUrl: undefined
+        }
+      )
+
+      )),
+
+  getInstitutionProducts: (institutionId: string): Promise<Array<ProductSwitchItem>> =>
+    apiClient
+      .get<Array<Product>>(GET_INSTITUTION_PRODUCTS(institutionId))
+      .then((response: AxiosResponse<Array<Product>>) => response.data.map((product) => (
+        {
+          id: product.id,
+          title: product.title,
+          productUrl: product.urlBO,
+          /* TODO */
```

Comment:
We added TODO because we don't get "linkType" property from be, so we used "internal" as value by default. In Interop fe they use the same value for that property.